app-web-frontend/src/MainComponents/Chats/ChatsPage.js

```diff
@@ -62,52 +63,83 @@ function ChatsPage() {
       },
       messages: [
         {
-          from: 3, // userID of sender
-          time: "11:04",
-          text: "Hello!"
+          senderId: 3, // userID of sender
+          created: "11:04",
+          content: "Hello!"
         },
         {
-          from: 3, 
-          time: "11:05",
-          text: "How are you?"
+          senderId: 3,
+          created: "11:05",
+          content: "How are you?"
         },
         {
-          from: 0, // my userID
-          time: "11:10",
-          text: "Good morning! Fine, thank you, Radjesh!"
+          senderId: 0, // my userID
+          created: "11:10",
+          content: "Good morning! Fine, thank you, Radjesh!"
         }
       ]
     }
   ]
+
   let [position, setPosition] = useState(0);
```

Comment:
nit: A bit controversial solution. Better approach would be to have 2 separate components, when we are in chat and when we are in chat list.

Reply:
Those already are separate components. Or do you mean the interface should be structured in another way?